source/api_documentation/transactions_api/_custom_data_params.rst

```diff
@@ -0,0 +1,34 @@
+
+
+Custom Data Parameters
+**********************
+
+Please click  `this link <https://www2.invoca.net/customer_data_dictionary/home>`_ to access the custom data parameters specific to your account.
+
+Custom Data & Signal fields can be accessed in the response by specifying the special `$invoca_custom_columns` constant in your `include_columns` parameter, or mentioning specific fields that you want to include, e.g. `include_columns=transaction_id,start_time_local,utm_medium,gclid,Quote,Purchase%20Made`.
```

Comment:
TIL that you could specify the partner names..

Reply:
I had to confirm in postman, but yup!